src/App.js

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import './App.scss';
+import Books from './pages/Books';
+import NavBar from './pages/NavBar';
+import NotMatch from './pages/NoMatch';
 
 function App() {
-  return <div className="app">App</div>;
+  return (
+    <Router>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="*" element={<NotMatch />} />
```

Comment:
- [ ] I believe this is an optional component for any route that does not match right?.  You will also require a `<Route>` for the `Categories` component and should display only a button with the text "Check status" as the requirement says, then you can optionally add this component(NotMatch) for any other route which is also important :+1:. It's vital for us to always stick to project requirements before including additional features.